pkg/didcomm/protocol/didexchange/service_test.go

```diff
@@ -1037,3 +1037,72 @@ func Test_AutoExecute(t *testing.T) {
 	close(ch)
 	<-done
 }
+
+func TestService_Errors(t *testing.T) {
```

Comment:
Why the underscore in the method name?

Reply:
[Removed](https://github.com/hyperledger/aries-framework-go/compare/47f4829dfae7a3436fed2eeeedd86507dfe297c4..f203a4aa73d5395f36c877dc61225060d87e0bea#diff-36ac39bebf32030fd7dff6f12aea8d7eR1041).